app/controllers/courses_controller.rb

```diff
@@ -237,6 +239,21 @@ def wiki_params
       .permit(:language, :project)
   end
 
+  def courses_wikis_params
+    params
+      .require(:course)
+      .permit(wikis: [:language, :project])
+  end
+
+  def update_courses_wikis
+    multi_wikis = courses_wikis_params[:wikis]
+    return if multi_wikis.nil?
```

Comment:
In what circumstances would mulit_wikis be nil?

Reply:
The course_params have partial parameters and wikis may not always be present. It was the case in courses_controller_spec.rb